modules/ROOT/pages/tls-configuration.adoc

```diff
@@ -93,12 +93,18 @@ Follow these steps to generate a keystore and export a self-signed certificate.
 For example, run the following command:
 +
 ----
-keytool -genkey -alias <key-alias> -keystore <keystore-name>.jks
+keytool -genkey -keyalg RSA -alias <key-alias> -keystore <keystore-name>.jks
 ----
 +
```

Comment:
D'oh! I missed this! I am so sorry!

Reply:
Remainder of option:
You must specify an encryption algorithm. If your configuration requires ECDSA, replace `-keyalg RSA` with `-keyalg EC`, but under no circumstances fail to specify one of these two values. If you do, the tool uses DSA by default, which is not compatible with TLS 1.2 and causes the SSL handshake to fail when the Mule app receives an HTTPS request.

Reply:
^ This is just an example of how narrative can replace the admonition.
But I think the choice to use an admonition is yours, based on your superior knowledge of what your audience understands and is likely to do or has done (resulting, perhaps, in past support cases).